bin/data-push.js

```diff
@@ -61,7 +61,14 @@ Promise.resolve().then(async () => {
         console.log('Error: You can push only local datasets.')
         process.exit(0)
       }
-      dataset = await Dataset.load(filePath)
+      try {
+        dataset = await Dataset.load(filePath)
+      } catch(err){
+        console.error('Error: '+err.message)
+        info("You can run 'data init <path>' to create a datapackage.")
```

Comment:
As you can see here https://github.com/datahq/data-cli/blob/master/bin/data-init.js "init" command doesn't consider <path> - you need to run it from the directory where you want to create a datapackage (so <path> is always current working dir).

Let's change it to:

```javascript
info("You can run 'data init' to create a `datapackage.json` from your project's root directory.")
```